CONTRIBUTING.md

```diff
@@ -51,10 +51,24 @@ asking for suggestions on how to address the documentation part.
 
 ## How to build and get up and running
 
+### Build Dependencies ###
+- [Go](https://golang.org/doc/install)
+- [Dep](https://github.com/golang/dep)
+
+### Getting the Repository ###
+
+`$ git clone https://github.com/fnproject/fn.git $GOPATH/src/github.com/fnproject/fn`
```

Comment:
pedantic: `go get github.com/fnproject/fn` would be the canonical way of doing this